### PR TITLE
[ISSUE #151] Adjust the categories sequence of sidebar and optimize SDK docs

### LINKED ISSUE #151: [Doc] Adjust the categories sequence of sidebar and optimize SDK docs
- Adjust the 'Quick Start' category to the top of sidebar. (https://github.com/apache/eventmesh-site/pull/132#discussion_r1322379468)
- Translate and optimize SDK docs.

### PR DESCRIPTION
**Pull request by Pil0tXia**
Fixes #151.

- Adjust the 'Quick Start' category to the top of sidebar. (https://github.com/apache/eventmesh-site/pull/132#discussion_r1322379468)
- Translate and optimize SDK Demo docs.

### CODE REVIEW

**Review by xwm1992 — APPROVED** [review]

File: docs/instruction/03-runtime.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # EventMesh Runtime
 
-The EventMesh Runtime is a stateful mesh node in an EventMesh cluster that is responsible for event transfer between the Source Connector and the Sink Connector, and can use EventMesh Storage as a storage queue for events.
+The EventMesh Runtime is a stateful mesh node in an EventMesh cluster that is responsible for event transfer between the Source Connector and the Sink Connector, and can use Event Store as a storage queue for events.
```
[the remark this answers is not in the document]
Comment (Pil0tXia) [reply docs/instruction/03-runtime.md:3]: According to the existing architecture diagram and usage in the documentation, the term "Event Store"（事件存储） is the most common and accurate usage. Following that is "EventMesh Store" or "EventMesh Storage"（EventMesh的存储模块）. I will standardize these usages.

File: docs/instruction/03-runtime.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
 ![EventMesh Runtime](/images/design-document/runtime.png)
```
[the remark this answers is not in the document]
Comment (Pil0tXia) [reply docs/instruction/03-runtime.md:5]: No problem, this is also a task in my plan. The links to these images are absolute paths based on the Docusaurus website, and they cannot be accessed correctly when editing the markdown files locally. Using relative paths can satisfy both online publishing and local editing needs. Additionally, relative paths pointing to markdown files can be correctly interpreted as web paths. I will standardize these links and update the contribution guidelines.

However, relative paths also have drawbacks. When a folder is moved, all relative paths in the markdown files will become invalid and need to be updated. Moreover, when there are many folder levels, the use of `../` can make it challenging to determine the hierarchical relationships.

---

没问题，这也是在我的计划中的一项任务。这些图像的链接是以docusaurus网站为基础的绝对路径，在本地编辑md文件时无法正确访问图片。使用相对路径即可同时满足线上发布和本地编辑的需要，同时，指向md文件的相对路径也可以被正确解析为web路径，这些链接我也会去统一，并更新贡献指南。

但相对路径也有缺点。当文件夹被移动时，所有的md文件中的相对路径都将失效，需要记得修改。而且，当文件夹层级较多时，`../`这种用法也会很多，会难以判断上下级关系。

File: docs/instruction/05-demo.md [hunk 3]
```diff
@@ -42,8 +42,11 @@ Run the main method of org.apache.eventmesh.tcp.demo.pub.eventmeshmessage.AsyncP
 
 More information about EventMesh-TCP, please refer to [EventMesh TCP](../sdk-java/03-tcp.md)
 
+## 2 HTTP
 
-## 2 HTTP DEMO
+>For HTTP, the eventmesh-sdk-java implements sending and subscribing to asynchronous events.
+>
+>In the demo, the `content` field of the Java class `LiteMessage` represents a special protocol. Therefore, if you are using the eventmesh-sdk-java's http-client, you only need to design the content of the protocol and provide the consumer's application at the same time.
```
[the remark this answers is not in the document]
Comment (Pil0tXia) [reply docs/instruction/05-demo.md:49]: Yes, it has been renamed in https://github.com/apache/eventmesh/pull/600. I will fix it.

File: docs/instruction/03-runtime.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Pil0tXia) [reply docs/instruction/03-runtime.md:5]: I do not recommend using the raw.githubusercontent.com domain to reference images. The URLs for these images have poor readability and are not conducive to file management. Additionally, githubusercontent is an unreliable image hosting service, as a large number of resources on GitHub reference this domain. Consequently, the CDN for this domain has a very short cache time, resulting in slow loading speeds within China and the potential for being blocked.

---

我不推荐使用raw.githubusercontent.com域名引用图片，其图片url可读性差，不利于管理文件。而且githubusercontent是一个不可靠的图床，GitHub上大量的资源都引用自这个域名，因此此域名的CDN对资源的缓存时间很短，在国内的加载速度慢，存在被封锁的可能性。

**Pull request merged**